spec/channels/conversation_channel_spec.rb

```diff
@@ -0,0 +1,145 @@
+require 'rails_helper'
+
+describe ConversationChannel, type: :channel do
+  let(:user)          { create(:user_with_conversations, conversations_count: 1) }
+  let(:conversation)  { user.conversations.last }
+  let(:message)       { 'Hello, Rails!' }
+  let(:params) do
+    {
+      conversation_id: conversation.id
+    }
+  end
+
+  describe 'with valid params' do
```

Comment:
This should be a context instead of a describe block

Reply:
Move this before block to the top of the context instead of each context below, since is the same for each example

```
before do
        stub_connection(current_user: user)
        subscribe(params)
  end
```